enterprise_extensions/blocks.py

```diff
@@ -124,6 +129,12 @@ def red_noise_block(psd='powerlaw', prior='log-uniform', Tspan=None,
     if psd in ['powerlaw', 'powerlaw_genmodes', 'turnover',
                'tprocess', 'tprocess_adapt', 'infinitepower']:
         # parameters shared by PSD functions
+        if logmin is not None and logmax is not None:
+            if prior == 'uniform':
+                log10_A = parameter.LinearExp(logmin, logmax)
+            else:
+                log10_A = parameter.Uniform(logmin, logmax)
+
```

Comment:
Won't your choices of logmin and logmax be just be overwritten by the next lines of code? i.e. when `prior` is set to its default value, this will reset what you have just set.

Reply:
You're right... I was just trying to figure out why the manual priors weren't working. Commit+push incoming!